atlante/sheet.go

```diff
@@ -62,6 +65,45 @@ type Sheet struct {
 	UseCached bool
 }
 
+func loadTemplateDir(t *template.Template, location *url.URL) (*template.Template, error) {
```

Comment:
please document that this function specifically looks for the directory "templates" (I saw this in the README).